project/Build.scala

```diff
@@ -915,34 +917,39 @@ object Build {
       unmanagedSourceDirectories in Test +=
         baseDirectory.value.getParentFile.getParentFile / "shared/src/test/scala",
 
-      if (isGeneratingForIDE) {
-        unmanagedSourceDirectories in Test +=
-          baseDirectory.value.getParentFile.getParentFile / "shared/src/test/scala-ide-stubs"
-      } else {
-        sourceGenerators in Test += Def.task {
+      BuildInfoPlugin.buildInfoScopedSettings(Test),
+      BuildInfoPlugin.buildInfoDefaultSettings,
```

Comment:
Seems like buildinfo does not deal well with the test configuration. I'm wondering if we should upstream this.

Reply:
At least, it would be worth file an issue, to discuss what can be done about it in the upstream plugin. It's not clear to me how a user of the plugin would be able to discard its effects for `Compile`, for example.

Reply:
Filed as: https://github.com/sbt/sbt-buildinfo/issues/186

>  It's not clear to me how a user of the plugin would be able to discard its effects for Compile, for example.

Do not enable the plugin, but just add the settings.